src/GraphqlMockingService.ts

```diff
@@ -6,6 +6,7 @@ type GraphqlMockingServiceOptions = {
   port?: number;
   subgraph?: boolean;
   graphQLIDE?: GraphQLIDE;
+  corsOptions?: any; // eslint-disable-line
```

Comment:
Based on how things are currently typed in the @types/cors library, we can't actually use the types in private methods. This is a bug in the types implementation, so for now we are using `any`.